File: java/src/main/java/org/training/speedrun/stage1/Stage1.java
package org.training.speedrun.stage1;

/**
 * Uncomment the tests, and make them pass
 */
public class Stage1 {

    enum Items {
        POTATO(1),
        TOMATO(2),
        LETTUCE(2),
        APPLE(1),
        BANANA(3),
        PEER(2);

        private final int price;

        Items(int price) {
            this.price = price;
        }

        public int getPrice() {
            return price;
        }
    }
}

File: kotlin/src/main/kotlin/org/training/speedrun/stage1/Stage1.kt
package org.training.speedrun.stage1

/**
 * Uncomment the tests, and make them pass
 */
class Stage1 {

    enum class Items private constructor(val price: Int) {
        POTATO(1),
        TOMATO(2),
        LETTUCE(2),
        APPLE(1),
        BANANA(3),
        PEER(2)
    }
}
